packages/fxa-content-server/tests/server/amplitude.js

```diff
@@ -2391,8 +2391,7 @@ registerSuite('amplitude', {
 
       assert.equal(logger.info.callCount, 1);
       const arg = logger.info.args[0][1];
-      assert.equal(arg.event_type, 'fxa_connect_device - view');
-      assert.equal(arg.event_properties.connect_device_flow, 'pairing');
```

Comment:
'screen.pair' is now explicitly matched; it was previously matched through a fuzzy event, which would then call `mapConnectDeviceFlow` with the "event category" to get this event property.

Reply:
This change would also stop our metrics from reporting this value (which I think is fine). I don't recall any dashboard that uses it.

Reply:
Are the Looker dashboards the only things using these events at this point?